glean-core/src/metrics/ping.rs

```diff
@@ -248,6 +260,17 @@ impl PingType {
                     ping.name
                 );
 
+                if !ping.schedules_pings.is_empty() {
+                    log::info!(
+                        "The ping '{}' is being used to schedule other pings: {:?}",
+                        ping.name,
+                        ping.schedules_pings
+                    );
+                    for scheduled_ping_name in &ping.schedules_pings {
+                        glean.submit_ping_by_name(scheduled_ping_name, reason);
```

Comment:
Hm, that means we also use the same reason. The reason for the actual submitted ping might not be specified for the linked ping.
We should at the very least document that related pings should use the same reasons (this might be done in a followup).